lua/colorizer.lua

```diff
@@ -305,7 +305,10 @@ do
       end
       local r, g, b, match_end = line:sub(i):match "^rgb%(%s*(%d+%%?)%s*,%s*(%d+%%?)%s*,%s*(%d+%%?)%s*%)()"
       if not match_end then
-         return
+         r, g, b, match_end = line:sub(i):match "^rgb%(%s*(%d+%%?)%s+(%d+%%?)%s+(%d+%%?)%s*%)()"
+         if not match_end then
+            return
+         end
```

Comment:
Doing an additional match here is unnecessary. Lua patterns support optional matching for fixed length patterns (see the `?` modifier).

Just add `?` after the commas in the first `match()` and it covers both cases:

```lua
local r, g, b, match_end = line:sub(i):match "^rgb%(%s*(%d+%%?)%s*,?%s*(%d+%%?)%s*,?%s*(%d+%%?)%s*%)()"
```

Reply:
Using the Question mark operator allows for mixing both seperators. 
```
rgb(20 20, 20) -- would be highlighted but not valid

rgb(20, 20 20)  -- same here

------
-- only these are valid
rgb(20 20 20)  
rgb(20, 20, 20)  
```

However CSS does not allow mixed statements, so sadly the simple question mark does not work the same way as a second `if` statement.